NeZha-D1s/Ubuntu/README.md

```diff
@@ -0,0 +1,65 @@
+# Ubuntu 23.10 VisionFive 2 版本测试报告
```

Comment:
23.10 -> 24.04?

Reply:
VF2 doesn't have the test result, I just created a README prepare for the new result,I'll change it and point out that still no result